corehq/motech/repeaters/models.py

```diff
@@ -212,6 +218,10 @@ class Meta:
             models.Index(fields=['domain']),
             models.Index(fields=['repeater_id']),
         ]
+        constraints = [
+            models.UniqueConstraint(fields=['repeater_id'],
+                                    name='one_to_one_repeater')
+        ]
```

Comment:
why isn't this done using [unique](https://docs.djangoproject.com/en/3.2/ref/models/fields/#unique)?